modules/migration-compatibility-guidelines-3-to-4.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/installing-3-4.adoc
+// * migrating_from_ocp_3_to_4/installing-restricted-3-4.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="migration-compatibility-guidelines-3-to-4_{context}"]
+= Compatibility guidelines
+
+You must install the {mtc-full} ({mtc-short}) Operator that is compatible with your {product-title} version.
+
+.Definitions
+
+control cluster:: The cluster that runs the {mtc-short} controller and GUI.
+remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
+
+You must use the compatible {mtc-short} version for migrating your {product-title} clusters. For the migration to succeed both your source cluster and the destination cluster must use the same version of MTC.
+
+MTC 1.7 supports migrations from {product-title} 3.11 to 4.9.
+
+MTC 1.8 only supports migrations from {product-title} 4.10 and later.
+
+.{mtc-short} compatibility: Migrating from {product-title} 3 to 4
+|===
+|Details |{product-title} 3.11 |{product-title} 4.12 or later
+
+|Stable {mtc-short} version
+|{mtc-short} v.1.7._z_
+|{mtc-short} v.1.8._z_
+
+|Installation
+|As described in this guide
+|Install with OLM, release channel `release-v1.8`
+|===
+
+Edge cases exist in which network restrictions prevent {product-title} 4 clusters from connecting to other clusters involved in the migration. For example, when migrating from an {product-title} 3.11 cluster on premises to a {product-title} 4 cluster in the cloud,  the {product-title} 4 cluster might have trouble connecting to the {product-title} 3.11 cluster. In this specific case, it is possible to designate the {product-title} 3.11 cluster as the control cluster and push workloads to the remote product-title} 4 cluster.
```

Comment:
Simple word `warning` in Vale

```suggestion
Edge cases exist in which network restrictions prevent {product-title} 4 clusters from connecting to other clusters involved in the migration. For example, when migrating from an {product-title} 3.11 cluster on premises to a {product-title} 4 cluster in the cloud,  the {product-title} 4 cluster might have trouble connecting to the {product-title} 3.11 cluster. In this specific case, it is possible to choose the {product-title} 3.11 cluster as the control cluster and push workloads to the remote product-title} 4 cluster.
```

Reply:
Going to stick with my wording here because I think "designate" better indicates that we are replacing or altering the usual choice. But in general, agree that a simpler word is often better.